examples/tpu_training-tf.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Love it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>